test/e2e/specs/wp-calypso-gutenberg-upgrade-spec.js

```diff
@@ -198,16 +119,6 @@ function verifyBlockInEditor( blockClass ) {
 	step( 'Block does not error in the editor', async function () {
 		await assertNoErrorInEditor();
 	} );
-
-	// Commented-out for now because of https://github.com/Automattic/jetpack/issues/16514
-	// step( 'Blocks do not invalidate', async function () {
-	// 		await assertNoInvalidBlocksInEditor();
-	// } );
-
-	// Commented out for now because of ENOMEM erros on CI
```

Comment:
Good house-keeping 👍🏻  I don't think those screenshots were ever useful. Plus, we already have hooks to take a screenshot/video when tests fail. I think these would be ideally replaced by automated visual regression tests.